src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -138,10 +138,19 @@ static bool _translucentRectAlphaMask(SwSurface* surface, const SwBBox& region,
     for (uint32_t y = 0; y < h; ++y) {
         auto dst = &buffer[y * surface->stride];
         auto cmp = &cbuffer[y * surface->stride];
-        for (uint32_t x = 0; x < w; ++x) {
-            auto tmp = ALPHA_BLEND(color, surface->blender.alpha(*cmp));
-            dst[x] = tmp + ALPHA_BLEND(dst[x], 255 - surface->blender.alpha(tmp));
-            ++cmp;
+        if (method == CompositeMethod::AlphaMask) {
```

Comment:
This if condition line is called every time during the for loop
The logic of this change is not wrong. But in tvgSwRaster, we don't like this because we want it to run as fast as possible. So, we create and use functions according to each rasterizing method.

How about this?
_translucentRectAlphaMask
_translucentRectInvAlphaMask